willow/plugins/pillow.py

```diff
@@ -63,6 +64,33 @@ def resize(self, size):
     def crop(self, rect):
         return PillowImage(self.image.crop(rect))
 
+    @Image.operation
+    def rotate(self, angle=90):
```

Comment:
I think we should remove the default value for the angle.

Reply:
No worries - will update this later today.